### DISCUSSION TITLE
Game Switcher isn't showing my playtime

### DISCUSSION THREAD

**Discussion by megusta420** [OnionUI/Onion#1599] [Support] [answered]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Docs](https://onionui.github.io/docs) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.3.0

### Miyoo Firmware Version

202306282128

### Provide a clear and concise description of the issue

Hey folks!

First, thanks for the awesome OS and support! I've been using my MMP with Onion for over a month now, and everything's been fantastic!

However, yesterday I noticed that my Game Switcher isn't showing my playtime for games.

I've tried clearing the cache, resetting settings, and removing/renaming the playActivity.db file, but none of that has worked. Onion just recreates the database, and the playtime information still isn't displayed.

One more thing I found strange: the /Saves/CurrentProfile/saves/PlayActivityBackup/ folder is missing.

### Steps to Reproduce

I don't know how to reproduce it, is just happened suddenly.

### Provide any additional context or information that may help us investigate your issue

![photo_2024-05-29 15 52 19](https://github.com/OnionUI/Onion/assets/28196153/9c37c790-eaac-422a-8b48-b94309fbff3a)

**Comment by Aemiii91** [comment, marked as answer]
The supplied screenshot shows GameSwitcher with the top bar hidden. This is toggled by pressing <kbd>Y</kbd>.

**Reply by megusta420** [reply]
Oh my dear Lord you're a lifesaver! Indeed, that's the solution! By the way why don't I have /Saves/CurrentProfile/saves/PlayActivityBackup/ folder, is it deprecated?

**Reply by Aemiii91** [reply]
Yes exactly, it's deprecated
